velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -129,5 +128,26 @@ TEST_F(StringAggregatesTest, nonFixedWidthAggregate) {
   testWindowFunction(input, "max(c2)", kOverClauses);
 }
 
+class CountAggregateTest : public WindowTestBase {};
```

Comment:
Please write a comment indicating why these are special test cases.

Also  I expected you to generate special result vectors for these. Is there something subtle I'm missing ?

Reply:
Sure, will add the comment. I just removed the empty frame tests for count aggregate here, would it not be better to add e2e tests in presto-native for empty frames?

Reply:
Lets add some basic tests here also.